mlflow/genai/experimental/databricks_trace_archival.py

```diff
@@ -394,9 +451,10 @@ def _do_enable_databricks_archival(
         ) from e
 
 
+# TODO: update experimental version number before merging
 @experimental(version="3.2.0")
 def enable_databricks_trace_archival(
-    experiment_id: str, catalog: str, schema: str, table_prefix: str = "trace_logs"
+    experiment_id: str, catalog: str, schema: str, table_prefix: str
```

Comment:
Not related to this PR but I do feel like this API name is confusing, if this is not user-facing anymore can we convert it to an internal API?